project-description.md

```diff
@@ -0,0 +1,21 @@
+
+The approach the issue, we first conduct a technical market analysis. We examine existing AI models on the market that can be used for text and image generation. In addition, we are to evaluate which models can be used for the detection of AI-generated content and assess their performance. In the second step, we develope completely new tool for AI content. The idea is to develop a program that reliably tricks existing AI detectors with little effort by making minimal changes to AI-generated text or images. This can then be used during training of a new model for augmenting data to robustly train a new model against such modifications. For this reason, the software we develop shall be fast and reliable on images of arbitrary size and text of arbitrary length.
+
+### Task
+The task consists of two parts. In the first part we conduct a technical market analysis, where we investigate existing models that can be used for text and image and image generation, as well as which models and algorithms can be used for the detection of of generated content. In the second part, we will develop a program that, by modifying AI-generated text and images as minimally as possible, can outsmart existing AI detectors by making minimal changes to AI-generated text and images.
+
+#### Technical market analysis
+The technical analysis will research generators for content generation by AI and programs for the detection of AI-generated content. The evaluation of the existing landscape for generative models should provide an overview of a domain that is developing very quickly and dynamically. The broadest possible range of software that can be used productively today should be considered, as well as an outlook on the latest research in this discipline. In addition to the analysis of generative models, we examine programs for the detection of AI-generated content and give an overview of existing products and current research. We evaluate the quality of the text and images generated by the generative models and the quality of the detection of the regressive models. We use quantitative, qualitative or hybrid approaches to evaluate the performance of the software.
+
+#### Augmentation of data
+In the second step, we develop a software tool for augmenting AI-created content. First we describe our theoretical approach on which the software is based. Then we develop a program that changes the input as minimally as possible, so that an AI detector always recognizes the output as a human-created text or image. Finally, we evaluate the quality of the software in detail in this part as well.
```

Comment:
```suggestion
In the second step, we develop a software tool for augmenting AI-created content. First we describe our theoretical approach on which the software is based on. Then we develop a program that changes the input as minimally as possible, so that an AI detector always recognizes the output as a human-created text or image. Finally, we evaluate the quality of the software in detail in this part as well.
```

Reply:
I don't think we need the "on" here.

Reply:
> I don't think we need the "on" here.

I agree with Lukas, We already have it in the middle of the sentence